devlib/module/cgroups.py

```diff
@@ -207,13 +208,15 @@ def tasks_per_group(self):
             tasks[cg] = self.tasks_count(cg)
         return tasks
 
+
 class CGroup(object):
 
-    def __init__(self, controller, name, create=True):
+    def __init__(self, controller, name, create=True, set_timeout=10):
```

Comment:
The timeout is really required only for the `freezer` controller... it seems a bit overfilling to have general timeout defined here for each and every controller.

What about moving it to the `set(self, **attrs)` method below or, maybe even better, fix the only really broken client, which is the `freeze` method defined at the bottom of this module?
Maybe we can implement freeze/thaw operations as a dedicated shutil call, which will properly include the required timeout for this operation only, to be used by the freeze() method.